test/intl402/NumberFormat/constructor-roundingIncrement-invalid.js

```diff
@@ -26,14 +27,22 @@ assert.throws(RangeError, function() {
   new Intl.NumberFormat([], {roundingIncrement: 5001});
 }, '5001');
 
-assert.throws(RangeError, function() {
+// InitializeNumberFormat ( numberFormat, locales, options ) (...)
```

Comment:
I might put these into the `info:` key in the front matter. Although I tend to find that the `info` key is less useful because it goes stale the fastest, it is current practice. You could also just phrase the comment as prose, something like "roundingIncrement is only allowed for fraction digits rounding".

Same goes for the comment below.

Reply:
I decided to drop the comments, the description it's self-explanatory  :)